src/theme.ts

```diff
@@ -52,6 +53,10 @@ export const THEME = {
   warningColor: PALETTE.gold,
   errorColor: PALETTE.red,
   infoColor: PALETTE.gray4,
+
+  // Sizes
+  fontSize: '14px', // antd default
+  size: 'middle' as SizeType,
```

Comment:
Better extract to ensure the value is a `SizeType`, as is a risky typecast.

```ts
const size: SizeType = 'middle';
```

Reply:
Makes sense, changed that

Reply:
Typescript didn't accept this in the stories:

`TS2322: Type '"large"' is not assignable to type '"middle" | undefined'`

Weirdly shrinked the possible values of the union type variable.

I changed the solution a bit.

Reply:
Good, I like the new solution. At some point, we might switch to:

```ts
const THEME: Theme = {}

type Theme = {
  size: SizeType,
 ...
}
```

Reply:
Adapted it directly to your suggestion, as `Record<string, unknown>` led to some type issues.